packages/relative-values/src/components/View/Header.tsx

```diff
@@ -15,13 +15,15 @@ export const Header: FC<{
   const cluster = item.clusterId ? clusters[item.clusterId] : undefined;
   return (
     <CellBox header clickable={clickable}>
-      <div className="text-xs font-bold p-1">
+      <div className="text-sm p-1 font-semibold text-slate-800 flex">
         {cluster ? (
-          <div className="float-right px-0.5">
+          <div className="float-left px-0.5 pr-2 pt-1.5">
             <ClusterIcon cluster={cluster} />
           </div>
         ) : null}
+        <div>
```

Comment:
I assume this is intentional (removes float), but combined with `text-sm` it eats into horizontal space more than I'd like. Especially because we need to add some padding on the right too:
<img width="186" alt="Captura de pantalla 2023-03-20 a la(s) 14 44 44" src="https://user-images.githubusercontent.com/89368/226461618-9a3bf937-7331-48c9-954c-b3486b8faba2.png">